bundling/bundle-browser.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -eu
+
+# Cloudflare Workers don't support top-level await
+sed -i~ 's/^if (isDeno)/if (false)/' ../src/platform.deno.ts
+trap 'mv ../src/platform.deno.ts~ ../src/platform.deno.ts' EXIT
+
+echo "Removing old bundle directory"
+rm -rf bundle/out
+
+if [ -d "../out" ]; then
+  echo "Found node files. Skipping d2n step"
+else
+  echo "Running d2n"
+  deno task backport
+fi
+
+echo "Creating bundle folder"
+mkdir -p bundle/out
+
+echo "Generating bundle package.json"
+jq '. + {
+  "name": "@grammyjs/web",
+  "dependencies": {},
+  "scripts": {},
+}' <../package.json >bundle/package.json
+
+echo "Copying node files"
+cp -R ../out bundle/
+cp ../LICENSE bundle/
+cp ../README.md bundle/
+shopt -s globstar
+rm bundle/**/*.js
```

Comment:
Wouldn't it be better to explicitly copy over the files we need, rather than copying over everything and then deleting some specific parts of it?

Reply:
It would, but glob patterns are very sketchy and I couldn't get cp to reproduce the existing folder structure but ignore js files

Reply:
`rsync`?

Reply:
I haven't used it before. 
I'll look into it